guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
```

Comment:
Wait ... why did we add this here? There's changes to the login module otherwise.